source/getting-started/configuration.markdown

```diff
@@ -24,7 +24,7 @@ Under the **Official add-ons** section, you will find the **File editor** add-on
 Now let's make a change using the file editor: we are going to change the name, location, unit system, and time zone of your Home Assistant installation.
 
  - Click the folder icon in the top left of the file editor window to open the file browser sidebar.
- - Click the `configuration.yaml` file (in the `/config/` folder) to load it into the main file editor window.
+ - Click the `configuration.yaml` file in `homeassistant/` (which maps to `/config/`) folder to load it into the main file editor window.
```

Comment:
We should fix this in the add-on, not documentation.

I suggest to revert this and adjust the add-on to have the expectation of this location to match everything else we do.

../Frenck